humans/auth/forms.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import PasswordResetForm as ResetForm
+
+User = get_user_model()
+
+
+class PasswordResetForm(ResetForm):
+    def get_users(self, email):
+        """
+        Given an email, return matching user(s) who should receive a reset.
+
+        This allows subclasses to more easily customize the default policies
+        that prevent inactive users from resetting their password.
+        """
+        active_users = User._default_manager.filter(
+            **{
+                '%s__iexact' % User.get_email_field_name(): email,
+                'is_active': True,
+            })
+        import pprint; pprint.pprint(active_users)
```

Comment:
This line can be deleted or replaced by a proper log. I use it to check if an e-mail is registered for a given user (so I knew everything was right when handling new membership requests).